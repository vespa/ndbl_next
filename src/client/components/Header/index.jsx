import React from 'react';
import css from './style.scss';

const Index = () => (
  <header className={css.header}>
    <div className={css.header__opening}>
      <div className={`${css['header__opening__box--opacity']} ${css.header__opening__box} `} />
      <div className={css.header__opening__box}>
        <h1>Purchasing assistant</h1>
        <h2>Amsterdam, the Netherlands</h2>
      </div>
    </div>
  </header>
);

export default Index;
