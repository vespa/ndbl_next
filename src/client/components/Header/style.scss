
@import '../../../scss/_vars.scss';

// background-image: url('../assets/img/open.jpg');
.header {
  background-color: #46474a;
  background-attachment: fixed;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  height: 305px;
  &--form {
    background-image: url('../../../assets/img/open.jpg');
  }
  &__opening{
    position: relative;
    height: 175px;
    width: 100%;
    max-width: $max_width_container;
    margin: 0 auto;
    text-align: center;
    &__box {
      color: #fff;
      height: inherit;
      margin-top: 52px;
      position: absolute;
      width: 100%;
      &--opacity{
        opacity: 0.5;
        background-color: $color_primary;
        height: 194px;
      }
      h1{
        padding: 40px 0 0 0;
        margin: 0;
        font-size: 60px;
        text-transform: uppercase;
        font-weight: normal
      }
      h2 {
        margin: 0;
        font-size: 40px;
        text-transform: uppercase;
        font-weight: normal
      }
    }
  }
}