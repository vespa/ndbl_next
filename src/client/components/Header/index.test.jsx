import React from 'react';
import { shallow } from 'enzyme';
import Header from './index';

describe('[Component: Input]', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Header />);
  });
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
