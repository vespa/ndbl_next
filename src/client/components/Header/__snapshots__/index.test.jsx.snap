// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component: Header] should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header
    img="open"
    subtitle="test 2"
    title="test"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className={undefined}
      >
        <div
          className="undefined undefined "
        />
        <div
          className={undefined}
        >
          <h1
            className={undefined}
          >
            test
          </h1>
          <h2
            className={undefined}
          >
            test 2
          </h2>
        </div>
      </div>,
      "className": "
      undefined undefined
    ",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="undefined undefined "
          />,
          <div
            className={undefined}
          >
            <h1
              className={undefined}
            >
              test
            </h1>
            <h2
              className={undefined}
            >
              test 2
            </h2>
          </div>,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "undefined undefined ",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1
                className={undefined}
              >
                test
              </h1>,
              <h2
                className={undefined}
              >
                test 2
              </h2>,
            ],
            "className": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "test",
                "className": undefined,
              },
              "ref": null,
              "rendered": "test",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "test 2",
                "className": undefined,
              },
              "ref": null,
              "rendered": "test 2",
              "type": "h2",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className={undefined}
        >
          <div
            className="undefined undefined "
          />
          <div
            className={undefined}
          >
            <h1
              className={undefined}
            >
              test
            </h1>
            <h2
              className={undefined}
            >
              test 2
            </h2>
          </div>
        </div>,
        "className": "
      undefined undefined
    ",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="undefined undefined "
            />,
            <div
              className={undefined}
            >
              <h1
                className={undefined}
              >
                test
              </h1>
              <h2
                className={undefined}
              >
                test 2
              </h2>
            </div>,
          ],
          "className": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "undefined undefined ",
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h1
                  className={undefined}
                >
                  test
                </h1>,
                <h2
                  className={undefined}
                >
                  test 2
                </h2>,
              ],
              "className": undefined,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "test",
                  "className": undefined,
                },
                "ref": null,
                "rendered": "test",
                "type": "h1",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "test 2",
                  "className": undefined,
                },
                "ref": null,
                "rendered": "test 2",
                "type": "h2",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
